src/Sulu/Bundle/MediaBundle/Media/ImageConverter/Transformation/PasteTransformation.php

```diff
@@ -87,9 +87,9 @@ public function execute(ImageInterface $image, $parameters)
     /**
      * Create mask.
      *
-     * @param $maskPath
-     * @param $width
-     * @param $height
+     * @param string|\Imagine\File\LoaderInterface|mixed $maskPath the file path, a LoaderInterface instance, or an object whose string representation is the image path
```

Comment:
this should be the return value of `$this->fileLocator->locate($parameters['image'])`

```suggestion
     * @param string|array The full path to the file or an array of file paths
```